doc/manuals/taskvine/index.md

```diff
@@ -2444,50 +2455,6 @@ Save this file as `parsl_vine_example.py`. Running
 `python parsl_vine_example.py`
 will automatically spawn a local worker to execute the function call.
 
-In order to use the TaskVineExecutor with remote resources, you will need to create a configuration as shown below. Using the TaskVine Factory is the simplest way of deploying remote workers. Here a configuration for HTCondor is shown. It is necessary to include a `project_name` in the `TaskVineManagerConfig` in order for the Factory to find the manager.
```

Comment:
Is there a reason this parsl doc was dropped? @colinthomas-z80 is this needed?

Reply:
Probably a bad merge. Yes it is needed

Reply:
Sorry I think I made a mistake while copying and pasting around.